src/pages/Tab1.tsx

```diff
@@ -11,6 +23,13 @@ const Tab1: React.FC = () => {
         </IonToolbar>
       </IonHeader>
       <IonContent fullscreen>
+        {loggedIn ? <h1>logged in</h1> : <h1>Not Logged in</h1>}
```

Comment:
There's some inconsistencies in capitalization here, is this something worth fixing?

Reply:
Oh ye, That was just to test some things real quick. I'll remove it now